pom.xml

```diff
@@ -78,6 +79,13 @@
     <commons.spotbugs.plugin.version>4.7.1.1</commons.spotbugs.plugin.version>
     <commons.spotbugs.impl.version>4.7.1</commons.spotbugs.impl.version>    
     <biz.aQute.bndlib.version>6.2.0</biz.aQute.bndlib.version>
+    <!-- default checkerframework annotation processors and compiler arguments -->
+    <ap1>org.checkerframework.checker.interning.InterningChecker</ap1>

```

Comment:
`<ap>` is too short a mnemonic to be clear to a reader.  Even lengthening it to `<annoProc>` would be better.  I would go for `<annotationProcessor>` because it only appears here and the wordiness is not a problem.